### ISSUE TITLE
Random Order keeping first slide the same

### ISSUE THREAD

**Issue by MediaCrushUK** [kenwheeler/slick#2461]
Hi There,

I am working with your carousel, it's great and has worked a treat!

I am using a piece of code to randomize the slide and it is currently set to keep the last slide in order at the end of the slide show. I am trying to keep the first slide in order instead and was wondering if you had any insight as to a cure?

The code to randomize is as follows;

`$.fn.randomize = function (selector) {
    var $elems = selector ? $(this).find(selector) : $(this).children(),
        $parents = $elems.parent();

    $parents.each(function () {
        $(this).children(selector).sort(function (childA, childB) {
            // * Prevent last slide from being reordered
            if($(childB).index() !== $(this).children(selector).length - 1) {
                return Math.round(Math.random()) - 0.5;
            }
			
        }.bind(this)).detach().appendTo(this);
    });

    return this;
};
`

I am not completely au fait with this Javascript method and any  help anyone can give would be much appreciated!

Many Thanks,

JJ @ MC

**Comment by syberknight** [comment]
i vote to re-open this question, as i have the same need but can't find a good solution.
